platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/settings/MerkleDbSettings.java

```diff
@@ -46,7 +46,7 @@ public interface MerkleDbSettings {
      * <b>IMPORTANT: This can only be set before a new database is created, changing on an
      * existing database will break it.</b>
      */
-    long getInternalHashesRamToDiskThreshold();
+    long getHashesRamToDiskThreshold();
```

Comment:
I assume `JasperDbConfig` needs to be updated, too.
https://github.com/hashgraph/hedera-services/blob/develop/platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/jasperdb/config/JasperDbConfig.java

Reply:
Sure, will do. Thank you